pkg/api/user/store.go

```diff
@@ -69,6 +69,18 @@ func (s *userStore) create(request *types.APIRequest, schema *types.APISchema, d
 	return s.Store.Create(request, schema, data)
 }
 
+func (s *userStore) Update(request *types.APIRequest, schema *types.APISchema, data types.APIObject, id string) (types.APIObject, error) {
+	newData := data.Data()
+	passwordPlainText := newData.String(FieldPassword)
+	password, err := pkguser.HashPasswordString(passwordPlainText)
+	if err != nil {
+		return types.APIObject{}, apierror.NewAPIError(validation.ServerError, "Failed to encrypt password")
+	}
+	newData.Set(FieldPassword, password)
+	data.Object = newData
+	return s.Store.Update(request, request.Schema, data, id)
```

Comment:
We may want to return the updated object without the password field like L53?

Reply:
fixed